Clang-11 warning - class destructor marked `final`

The following warning is produced by Clang-11:
```
src/UI/MineOperationsWindow.h:16:34: warning: class with destructor marked 'final' cannot be inherited from [-Wfinal-dtor-non-final-class]
        virtual ~MineOperationsWindow() final;
                                        ^
```

The warning seems to be newly introduced by a compiler update.

--- a/src/UI/MineOperationsWindow.h
+++ b/src/UI/MineOperationsWindow.h
@@ -9,7 +9,7 @@
 /**
  * \brief Implements the Mine Facility Operations Window
  */
-class MineOperationsWindow : public Window
+class MineOperationsWindow final : public Window
 {
 public:
 	MineOperationsWindow();

